test/verifiedTests/testPrint/testPrintRxnFormula.m

```diff
@@ -37,8 +41,8 @@
 formulas = printRxnFormula(model, model.rxns, true, true, true, 1, true, true);
 diary off
 
-text1 = fileread('refData_printRxnFormulaGPR.txt');
-text2 = fileread('printRxnFormula.txt');
+text1 = importdata('refData_printRxnFormulaGPR.txt');
+text2 = importdata('printRxnFormula.txt');
```

Comment:
Why the changes from fileread to importdata? I know the main difference between them is that linebreaks get ignored but would this have any influence?